explore/Target_therapy.csv

```diff
@@ -0,0 +1,47 @@
+Drug,Entrez_gene_id,Gene_symbol

```

Comment:
Ok, I will change the file and also change or remove the accuracy score. I have added AUROC values, but the results are not encouraging. I will first try the balanced accuracy scores. Hopefully, this approach will work. 
Thanks.